extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmProcessor.java

```diff
@@ -1021,26 +1023,19 @@ private static void producePersistenceUnitDescriptorFromConfig(
             descriptor.getProperties().setProperty(AvailableSettings.GENERATE_STATISTICS, "true");
         }
 
-        // sql-load-script
+        // sql-load-scripts
         Optional<String> importFile = getSqlLoadScript(persistenceUnitConfig.sqlLoadScript, launchMode);
 
         if (importFile.isPresent()) {
-            Path loadScriptPath = applicationArchivesBuildItem.getRootArchive().getChildPath(importFile.get());
+            Boolean allLoadScriptFilesExist = Stream.of(importFile.get().split(","))
```

Comment:
You don't need to do the splitting yourself; I think you can just change the type of the `sqlLoadScript` property to `Option<List<String>>`. That would be more explicit, I believe?